src/middlewared/middlewared/plugins/vm/utils.py

```diff
@@ -4,6 +4,7 @@
 
 
 ACTIVE_STATES = ['RUNNING', 'SUSPENDED']
+FINAL_NVRAM_FOLDER_PATH = '/data/subsystems/vm/nvram'
```

Comment:
Why do we call it `FINAL`? This name is confusing outside of data migration context.